templates/bridgeserver2.yaml

```diff
@@ -0,0 +1,834 @@
+        # Application environment options
+        - Namespace: 'aws:elasticbeanstalk:application:environment'
+          OptionName: ENV
+          Value: !Ref Env
+        - Namespace: 'aws:elasticbeanstalk:application:environment'
+          OptionName: bridge.env
+          Value: !Ref BridgeEnv
+        - Namespace: 'aws:elasticbeanstalk:application:environment'
+          OptionName: bridge.user
+          Value: !Ref BridgeUser
+  AWSEBApplication:
+    Type: 'AWS::ElasticBeanstalk::Application'
+    Properties:
+      ApplicationName: !Join
+        - '-'
+        - - !Ref 'AWS::StackName'
+          - application
+  AWSEBEnvironment:
+    Type: 'AWS::ElasticBeanstalk::Environment'
+    Properties:
+      ApplicationName: !Ref AWSEBApplication
+      TemplateName: !Ref AWSEBConfigurationTemplate
+      EnvironmentName: !Join
+        - '-'
+        - - !Ref 'AWS::StackName'
+      Tier:
+        Name: WebServer
+        Type: Standard
+  AWSS3AppDeployBucket:
+    Type: 'AWS::S3::Bucket'
+  AWSR53RecordSet:
+    Type: 'AWS::Route53::RecordSet'
+    Properties:
+      HostedZoneName: !Join
+        - ''
+        - - !Ref DNSDomain
+          - .
+      Name: !Join
+        - ''
+        - - !Ref DNSHostname
+          - .
+          - !Ref DNSDomain
+          - .
+      Type: CNAME
+      TTL: '900'
+      ResourceRecords:
+        - !GetAtt
+          - AWSEBEnvironment
+          - EndpointURL
+  AWSR53HealthCheck:
+    Type: "AWS::Route53::HealthCheck"
+    DependsOn: AWSR53RecordSet
+    Properties:
+      HealthCheckConfig:
+        FullyQualifiedDomainName: !GetAtt AWSEBEnvironment.EndpointURL
+        Port: "80"
+        Type: "HTTP"
+        ResourcePath: "/"
+      HealthCheckTags:
+        -
+          Key: "Name"
+          Value: !Join
+            - '-'
+            - - !Ref 'AWS::StackName'
+              - AWSR53HealthCheck
+  AWSIAMServiceRole:
+    Type: 'AWS::IAM::Role'
+    Properties:
+      AssumeRolePolicyDocument:
+        Version: 2012-10-17
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service:
+                - elasticbeanstalk.amazonaws.com
+            Action:
+              - 'sts:AssumeRole'
+            Condition:
+              StringEquals:
+                sts:ExternalId: elasticbeanstalk
+      Path: /
+      ManagedPolicyArns:
+        - 'arn:aws:iam::aws:policy/service-role/AWSElasticBeanstalkEnhancedHealth'
+        - 'arn:aws:iam::aws:policy/service-role/AWSElasticBeanstalkService'
+        - 'arn:aws:iam::aws:policy/service-role/AmazonEC2RoleforSSM'
+  AWSIAMRole:
+    Type: 'AWS::IAM::Role'
+    Properties:
+      AssumeRolePolicyDocument:
+        Version: 2012-10-17
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service:
+                - ec2.amazonaws.com
+            Action:
+              - 'sts:AssumeRole'
+      Path: /
+      ManagedPolicyArns:
+        - 'arn:aws:iam::aws:policy/AWSElasticBeanstalkWebTier'
```

Comment:
Since BridgeServer2 is intended to replace BridgePF, for now, I'm just using the same permissions. If we need to restrict the permissions further, we can do that as a later step.